src/skmultiflow/meta/adaptive_random_forest_regressor.py

```diff
@@ -0,0 +1,497 @@
+
+    lambda_value: int, optional (default=6)
+        The lambda value for bagging (lambda=6 corresponds to Leverage Bagging).
+
+    drift_detection_method: BaseDriftDetector or None, optional
+        (default=ADWIN(0.001))
+        Drift Detection method. Set to None to disable Drift detection.
+
+    warning_detection_method: BaseDriftDetector or None, default(ADWIN(0.01))
+        Warning Detection method. Set to None to disable warning detection.
+
+    max_byte_size: int, optional (default=33554432)
+        (`RegressionHoeffdingTree` parameter)
+        Maximum memory consumed by the tree.
+
+    memory_estimate_period: int, optional (default=2000000)
+        (`RegressionHoeffdingTree` parameter)
+        Number of instances between memory consumption checks.
+
+    grace_period: int, optional (default=50)
+        (`RegressionHoeffdingTree` parameter)
+        Number of instances a leaf should observe between split attempts.
+
+    split_confidence: float, optional (default=0.01)
+        (`RegressionHoeffdingTree` parameter)
+        Allowed error in split decision, a value closer to 0 takes longer
+        to decide.
+
+    tie_threshold: float, optional (default=0.05)
+        (`RegressionHoeffdingTree` parameter)
+        Threshold below which a split will be forced to break ties.
+
+    binary_split: bool, optional (default=False)
+        (`RegressionHoeffdingTree` parameter)
+        If True, only allow binary splits.
+
+    stop_mem_management: bool, optional (default=False)
+        (`RegressionHoeffdingTree` parameter)
+        If True, stop growing as soon as memory limit is hit.
+
+    remove_poor_atts: bool, optional (default=False)
+        (`RegressionHoeffdingTree` parameter)
+        If True, disable poor attributes.
+
+    no_preprune: bool, optional (default=False)
+        (`RegressionHoeffdingTree` parameter)
+        If True, disable pre-pruning.
+
+    leaf_prediction: string, optional (default='nba')
+        (`RegressionHoeffdingTree` parameter)
+        Prediction mechanism used at leafs.
+
+        - 'mc' - Majority Class
+        - 'nb' - Naive Bayes
+        - 'nba' - Naive Bayes Adaptive
+
+    nb_threshold: int, optional (default=0)
+        (`RegressionHoeffdingTree` parameter)
+        Number of instances a leaf should observe before allowing Naive Bayes.
+
+    nominal_attributes: list, optional
+        (`RegressionHoeffdingTree` parameter)
+        List of Nominal attributes. If emtpy, then assume that all attributes
+        are numerical.
+
+    learning_ratio_perceptron: float
+        (`RegressionHoeffdingTree` parameter)
+        The learning rate of the perceptron.
+
+    learning_ratio_decay: float
+        (`RegressionHoeffdingTree` parameter)
+        Decay multiplier for the learning rate of the perceptron
+
+    learning_ratio_const: Bool
+        (`RegressionHoeffdingTree` parameter)
+        If False the learning ratio will decay with the number of examples seen
```

Comment:
The default values should appear. Could you add them?

Reply:
I added the values from RegressionHoeffdingTree, could you quickly check please?